tia-client/src/main/java/com/teamscale/tia.client/CommandLineInterface.java

```diff
@@ -81,7 +81,13 @@ private void runCommand() throws Exception {
 	}
 
 	private void endTestRun() throws Exception {
-		AgentCommunicationUtils.handleRequestError(api::testRunFinished,
+		boolean partial;
+		if (arguments.size() == 1) {
+			partial = Boolean.parseBoolean(arguments.remove(0));
+		} else {
+			partial = false;
+		}
```

Comment:
The comment is a bit misformatted. I assume you meant:

```suggestion
		boolean partial = false;
		if (arguments.size() == 1) {
			partial = Boolean.parseBoolean(arguments.remove(0));
		}
```
right? I would have written it the same way normally, but this does not work here as the `partial` variable needs to be effectively final so that it can be captured in the lambda in the following line. If you would make the variable mutate the compiler will no longer accept it and you'd have to copy it into a `finalPartial` temp variable or similar, which is less readable than the current version IMO. Do you agree?

Reply:
Thanks for the explanation! I agree on that